src/components/Tabs/MainTab.tsx

```diff
@@ -150,6 +150,60 @@ const currentTab = css`
   top: 1px;
 `;
 
+const NotificationAddOn = styled.div`
+  position: absolute;
+  left: 13.5px;
+  top: 4px;
+  border: 2px solid #1f8ce6;
+  width: 11px;
+  height: 11px;
+  background: #ffde24;
+  border-radius: 11px;
+  background-clip: padding-box;
+  ${orBelow(BreakPoint.LG, css`
+    left: 17.5px;
+  `)};
+`;
+
+const CartAddOnWrapper = styled.div`
+  position: absolute;
+  justify-content: flex-end;
+  margin-left: auto;
+  width: 100%;
+  max-width: 30px;
+  align-items: center;
+  top: -10px;
+  left: 10px;
+  display: flex;
+  max-height: 31px;
+  height: 100%;
+  ${orBelow(BreakPoint.LG, css`
+      left: 12.5px;
```

Comment:
```suggestion
    left: 12.5px;
```

아랫줄이 지워졌더니 이게 outdated라고 뜨네요

Reply:
아이고 ㅠㅠ ide가 이상한건지 계속 못잡네요  🙇